src/content/topics/integrations/more/compass.mdx

```diff
@@ -0,0 +1,144 @@
+
+To connect a LaunchDarkly project to a Compass component:
+
+1. In LaunchDarkly, navigate to the **Feature flags** page for the project you want to connect.
+2. Copy the project URL, e.g. https://app.launchdarkly.com/starship-command/production/features, to your clipboard.
+3. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
+4. Navigate to the Compass component you want to connect to LaunchDarkly.
+5. In the **Other links** section of your Compass component, paste the LaunchDarkly project URL as a new link.
+6. Verify that the component was successfully connected by clicking on the **LaunchDarkly** tab in the left navigation.
+
+![Successfully connected Compass component to LaunchDarkly](compass-ld-integration-component-connected.png)
+
+7. Repeat the above steps to associate your other LaunchDarkly projects to the applicable Compass components.
+
+## Viewing feature flag information for a Compass component
+
+Once you have connected your Compass components to LaunchDarkly, you can view feature flag information on the component page in Compass.
+
+1. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
+2. Navigate to the Compass component for which you want to view feature flag information.
+3. Make sure you have completed the steps in [Connecting Compass components to LaunchDarkly](#connecting-compass-components-to-launchdarkly).
+4. Click on the **LaunchDarkly** tab in the left navigation.
+
+### Recent feature flag changes for a connected Compass component
+
+You can see a history of recent feature flag changes for the associated LaunchDarkly project on the **Recently updated** subtab.
+* Click the dot next to each item to see more details about the feature flag change.
+* Click the feature flag name to view the feature flag in LaunchDarkly.
+
+![Recently feature flag changes](compass-ld-integration-recently-updated.png)
+
+### All feature flags for a connected Compass component
+
+You can see a list of all feature flags for the associated LaunchDarkly project on the **All feature flags** subtab.
+* Use the **Filter by status** dropdown to filter the list of flags displayed in the list
+* Click the column headings above the list to sort by the **Flag**, **Status** or **Last requested** column.
+* Click the feature flag name to view the feature flag in LaunchDarkly.
+
+![All feature flags](compass-ld-integration-all-feature-flags.png)
+
+## Disconnecting Compass components from LaunchDarkly
+
+The connection between a LaunchDarkly project and a Compass component is stored as a link on the Compass component, as described in the [Connecting Compass components to LaunchDarkly](#connecting-compass-components-to-launchdarkly) section. If desired, you can remove this connection on the Compass component page.
+
+1. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
+2. Navigate to the Compass component you want to connect to LaunchDarkly.
```

Comment:
```suggestion
2. Navigate to the Compass component you want to disconnect from LaunchDarkly.
```
I think update fixes a copy-paste error, but please check this for accuracy! Thanks.